src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -25,14 +25,20 @@ struct event_entry_t {
   std::string push_endpoint_args;
   std::string arn_topic;
   ceph::coarse_real_time creation_time;
+  uint32_t persistency_time_to_live;
```

Comment:
please init to zero, for the case that we decode with ver2

Reply:
I put "-1" as the below comment suggested